test/langtools/jdk/jshell/CompletenessTest.java

```diff
@@ -99,8 +99,8 @@ public class CompletenessTest extends KullaTesting {
         "int p",
         "Deque<Token> stack = new ArrayDeque<>()",
         "final Deque<Token> stack = new ArrayDeque<>()",
-        "java.util.Scanner input = new java.util.Scanner(System.in)",
-        "java.util.Scanner input = new java.util.Scanner(System.in) { }",
+        "java.util.Scanner input = new java.util.Scanner(System.in, System.getProperty(\"stdin.encoding\"))",
+        "java.util.Scanner input = new java.util.Scanner(System.in, System.getProperty(\"stdin.encoding\")) { }",
```

Comment:
These seem to be checking a unit is complete or not, so these changes may be irrelevant too.